src/vivarium_public_health/population/add_new_birth_cohorts.py

```diff
@@ -176,9 +176,9 @@ def setup(self, builder):
         """
 
         self.randomness = builder.randomness.get_stream('fertility')
-        asfr_data = builder.data.load("covariate.age_specific_fertility_rate.estimate")[['year_start', 'year_end',
-                                                                                         'age_group_start',
-                                                                                         'age_group_end', 'value']]
+        asfr_data = builder.data.load("covariate.age_specific_fertility_rate.estimate")
+        asfr_data = asfr_data[asfr_data.sex == 'Female'][['year_start', 'year_end',
```

Comment:
GBD includes Males in its fertility rate data, although the values are all 0

Reply:
same here. Many of sex restricted (or age restricted) data often ends up having restricted age/sex with all 0 data :( (sometimes also have non-zero data)